### PR TITLE
Update Ingress resources to be compatible with k8s 1.22

### PR DESCRIPTION
**Pull request by ubergesundheit**
This PR

- Adds Ingress compatibility for clusters above Kubernetes < 1.22
- Change the type of the certificate secrets to kubernetes/tls. This enables monitoring of said certificate using cert-exporter

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Review by marians — APPROVED** [review]
LGTM, thank you for the service!

File: helm/athena/templates/ingress.yaml [hunk 1]
```diff
@@ -1,4 +1,8 @@
+{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress" }}
```
Comment (marians) [inline helm/athena/templates/ingress.yaml:1]: Do we actually need all these conditionals if we only want to run this on 1.19 and higher?

Comment (marians) [reply helm/athena/templates/ingress.yaml:1]: Hmm. I based my comment on "This makes the chart incompatible with Kubernetes < 1.19" ind the PR description and a changelog entry, which was outdated it seems.

Now I am confused whether or not the chart is meant to run on Kubernetes older than 1.19.

**Review by marians — APPROVED** [review]

File: helm/athena/templates/ingress.yaml (hunk 1 above)
Comment (ubergesundheit) [reply helm/athena/templates/ingress.yaml:1]: The Ingress apiVersion used in the latest version of athena is compatible with Kubernetes older than 1.19. This change makes it possible to install athena on clusters above 1.22 (See https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122)

Thanks for noticing and raising!

**Pull request merged**